src/Drivers/Sqlite.php

```
@@ -0,0 +1,123 @@
+use PDOException;
+use RuntimeException;
+use SessionHandlerInterface;
+
+class SQLite extends SetGet implements SessionHandlerInterface
+{
+    private PDO $conn;
+    private string $table;
+
+    public function __construct(array $config)
+    {
+        if (!extension_loaded('pdo_sqlite')) {
+            throw new RuntimeException('\'pdo_sqlite\' extension is needed to use this driver.');
+        }
+
+        parent::__construct($config);
+        $config = $config[Session::CONFIG_SQLITE_DS];
+        $dsn = "{$config['driver']}:{$config['db_path']}";
+        $this->table = $table = $config['db_table'];
+
+        try {
+            $this->conn = new PDO($dsn, null, null, [
+                PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+            ]);
+
+            // Enable Write-Ahead Logging (WAL) mode
+            $this->conn->exec('PRAGMA journal_mode = WAL');
+
+            $this->conn->query("SELECT 1 FROM `{$table}` LIMIT 1");
+        } catch (PDOException $e) {
+            // Debug information
+            error_log("Error connecting to SQLite database or checking table: " . $e->getMessage());
+
+            $this->conn->query('CREATE TABLE `' . $table . '` (
+                `id` TEXT PRIMARY KEY,
+                `data` TEXT NOT NULL,
+                `time` INTEGER NOT NULL
+            )');
+        }
+    }
+
+    public function open($path, $name): bool
+    {
+        return true;
+    }
+
+    public function close(): bool
+    {
+        return true;
+    }
+
+    public function read($id): string
+    {
+        $data = '';
+        $statement = $this->conn->prepare("SELECT `data` FROM `{$this->table}` WHERE `id` = :id");
+        $statement->bindParam(':id', $id, PDO::PARAM_STR);
+
+        if ($statement->execute()) {
+            $result = $statement->fetch();
+            $data = $result['data'] ?? '';
+        } else {
+            // Debug information
+            error_log("Failed to execute read statement for ID: {$id}");
```

Lets throw an exception instead of error_log